src/cart/crt_hg.h

```diff
@@ -52,6 +52,14 @@ crt_na_type_is_ofi(int na_type)
 	       (na_type < CRT_NA_OFI_COUNT);
 }
 
+static inline bool
+crt_na_type_is_slow(int na_type)
```

Comment:
i would prefer crt_provider_is_slow() similar to how we have other crt_provider_*() calls added for multi-provider support later on (e.g. crt_provider_is_contig_ep()), but its not a big deal since we have crt_na_type_is_ofi() as well and that will be renamed later as part of multi provider patch